src/main/java/ca/team2706/frc/robot/config/Config.java

```diff
@@ -363,4 +385,49 @@ public static XboxValue getXboxValueFromNTKey(final String ntKey) {
          */
         POV
     }
+
+    /**
+     * Gets the value of TRAJ_DELTA_TIME.
+     *
+     * @return The value of TRAJ_DELTA_TIME.
+     */
+    public static double getTRAJ_DELTA_TIME() {
+        return TRAJ_DELTA_TIME.value();
```

Comment:
I don't think it's actually necessary to have a method for accessing these fluid constants (this one and the methods below this too). We usually just call `FluidConstant.value()` wherever it is we want the value.

Reply:
I agree that these method seem extraneous. They were added purely to be able to fix certain values in the unit/regression tests. (The motivation for this is in the comment about partially mocking Config above.) Can you suggest an alternate way of doing this in the tests?

Reply:
Since all mocking of Config has now been removed, these methods have been eliminated.